Assets/Projectile.cs

```diff
@@ -28,12 +30,13 @@ void FixedUpdate()
     void OnTriggerEnter2D(Collider2D other)
     {
         Health health = other.gameObject.GetComponent<Health>();
+        Team otherTeam = other.GetComponent<Team>();
 
-        // check whether the colliding object can be damaged
-        if (health)
+        // check whether the colliding object can be damaged and can not harm objects that are on the same team
+        if (health && otherTeam.GetTeam() != this.team.GetTeam())
```

Comment:
We should also do a check for otherTeam not being null `if (health && team && ...)`